File: Fall-2015/ACTs/MATLAB-2/task3.m
% Activity: MATLAB 2 Bonus, Task 3
% File: ML2_ACT_Task3_katherto.m
% Date: 22 November 2015
% By: Kathryn Atherton
% katherto
% Ryan Hellyer
% rhellyer
% Natalie Zimmermann
% zimmermn
% 
% Section: 04
% Team: 59
%
% ELECTRONIC SIGNATURE
% Kathryn Atherton
% Ryan Hellyer
% Natalie Zimmermann
%
% The electronic signatures above indicate that the script
% submitted for evaluation is the combined effort of all
% team members and that each member of the team was an
% equal participant in its creation. In addition, each
% member of the team has a general understanding of all
% aspects of the script development and execution.
%
% A BRIEF DESCRIPTION OF WHAT THE SCRIPT OR FUNCTION DOES
% This function plots the function sin(x) with various 
% steps of x values. 

% clear the command window of previous variables and functions
clear
clc

% set the values for x
x = 0:2:10;

% set the function for y
y = sin(x);

% graph the function y vs x
plot(x, y)

%% new section

% plot on the same graph
hold on

% set the values for x
x = 0:1:10;

% set the function for y
y = sin(x);

% graph the function y vs x
plot(x,y)


%% new section

% set the values for x
x= 0:0.01:10;

% set the function for y
y = sin(x);

% graph the function y vs x
plot(x,y)

File: Fall-2015/ACTs/Python-2/task3.py
#Activity no. : Python 2 Activity

#File: Py2_ACT_Task3_katherto.py

#By: Natalie Zimmermann

#login: zimmermn

#By:Ryan Hellyer

#login:rhellyer

#By: Kathryn Atherton

#login: katherto

#Section: 04

#Team: Team 59

#
#ELECTRONIC SIGNATURE

#Natalie Zimmermann

#Ryan Hellyer

#Kathryn Atherton

#
#The electronic signature above indicates the program submitted for evaluation represents the work of all team  members and that all team  

#members have a general understanding of the program. 

#
#PROGRAM DESCRIPTION

#The following program has two user-defined functions, distanceRover that calculates 

#the distance a robot can drive in a given time, and solarPanelArea, that calculates

#the area needed for a specific efficency, power, and solar irradiance


speed = 1

time = 2

power = 100

efficency = 0.25

area = 0

distance = 0

solar_Irradiance = 590
 


time = time * 60 * 60
 


def distanceRover (speed, time, distance):
	
    distance = speed * time 
	
    return distance 



def solarPanelArea (power, efficency, area, distance, solar_Irradiance):
	
    area = power / (efficency * solar_Irradiance) 
	
    return area

print ('The robot travelled', distanceRover (speed, time, distance),' cm in', time, 
'seconds\nAnd the solar panel surface area is:',solarPanelArea (power, efficency, area, distance, solar_Irradiance),
'square meters')